srcpkgs/ergo/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'ergo'
+pkgname=ergo
+version=2.9.1
+revision=1
+build_style=go
+go_import_path="github.com/ergochat/ergo"
+#hostmakedepends="git"
```

Comment:
commenting this out doesn't make much sense, let's remove it

Reply:
Whoops, terribly sorry, wanted to do that but forgot